package.json

```diff
@@ -60,10 +56,14 @@
     "fp"
   ],
   "license": "Apache-2.0",
-  "repository": "netlify/eslint-config-node",
+  "repository": {
```

Comment:
These are automatically added by `npm init`. We can still just use `"repository": "netlify/eslint-config-node"` but the `bugs` is redundant. Let me know which one you prefer.